codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -387,7 +398,7 @@ public void onSettingsAckRead(ChannelHandlerContext ctx) throws Http2Exception {
         private void applyLocalSettings(Http2Settings settings) throws Http2Exception {
```

Comment:
Could we update the method comment to indicate that it is only called after the local settings have been acknowledged by the remote endpoint?